CollAction/Controllers/ProjectsController.cs

```diff
@@ -90,7 +90,7 @@ public ViewResult StartInfo()
         public async Task<IActionResult> Embed(int? projectId)
         {
             var projects = projectId.HasValue ? await DisplayProjectViewModel.GetViewModelsWhere(_context, p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted && p.Id == projectId.Value)
-                                              : await DisplayProjectViewModel.GetViewModelsWhere(_context, p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted);
+                                              : await DisplayProjectViewModel.GetViewModelsWhere(_context, p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted && p.Category.Name == "Friesland");
```

Comment:
Why the specific 'friesland' check? We could leave it up to the implementer to select the correct project id's, so this change can easily merge into the master branch as well.